app/middleware/auth.js

```diff
@@ -0,0 +1,37 @@
+import 'babel-polyfill';
+import jwt from 'jsonwebtoken';
+
+/**
+ * @function verifyToken
+ * @param {string} data - A token
+ * @returns { null } returns Unauthorized Access if token is undfefined
+ */
+
+const auth = (req, res, next) => {
```

Comment:
There is no comment for this function, also what is the difference between this function and the `verifyToken` function below it

Reply:
-Commented
-The verifyToken function is used to check if the user token is expired.
-It is not meant to be passed across controller unlike auth function that decodes the payload in the token and passes it to controllers